src/Tickets/Commerce/Gateways/PayPal/Buttons.php

```diff
@@ -55,6 +56,11 @@ public function get_checkout_script() {
 			return;
 		}
 
+		// Bail if there are no tickets in cart.
+		if ( empty( tribe( Cart::class )->get_items_in_cart( true ) ) ) {
```

Comment:
We can't use function into `empty()` needs to be saved into a variable to be tested.